internal/service/sso/resource_identity_provider_attribute_test.go

```diff
@@ -88,6 +88,44 @@ func TestAccIdentityProviderAttribute_RemovalDrift(t *testing.T) {
 	})
 }
 
+func TestAccIdentityProviderAttribute_MicrosoftIdentityProvider(t *testing.T) {
+	t.Parallel()
+
+	resourceName := acctest.ResourceNameGen()
+	resourceFullName := fmt.Sprintf("pingone_identity_provider_attribute.%s", resourceName)
+
+	name := resourceName
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			acctest.PreCheckNoTestAccFlaky(t)
+			acctest.PreCheckClient(t)
+			acctest.PreCheckNoFeatureFlag(t)
+		},
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		CheckDestroy:             sso.IdentityProviderAttribute_CheckDestroy,
+		ErrorCheck:               acctest.ErrorCheck(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIdentityProviderMicrosoftAttributeConfig(resourceName, name),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestMatchResourceAttr(resourceFullName, "id", verify.P1ResourceIDRegexpFullString),
+					resource.TestMatchResourceAttr(resourceFullName, "environment_id", verify.P1ResourceIDRegexpFullString),
+					resource.TestMatchResourceAttr(resourceFullName, "identity_provider_id", verify.P1ResourceIDRegexpFullString),
+					resource.TestCheckResourceAttr(resourceFullName, "name", "username"),
+					resource.TestCheckResourceAttr(resourceFullName, "update", "EMPTY_ONLY"),
+					resource.TestCheckResourceAttr(resourceFullName, "value", "${providerAttributes.email}"),
+					resource.TestCheckResourceAttr(resourceFullName, "mapping_type", "CORE"),
+				),
+			},
+			{
```

Comment:
Is this step necessary? The TF tests automatically run destroy after all steps are completed.